pkg/agent/agent.go

```diff
@@ -257,6 +269,51 @@ func registerCredentialProvider(idp config.IDPConfig, tlsCfg config.TLSConfig, p
 			WithField("metadata-url", idp.GetMetadataURL()).
 			Errorf("unable to register external IdP provider, any credential request to the IdP will not be processed. %s", err.Error())
 	}
+	crdName := idp.GetIDPTitle() + " " + provisioning.OAuthIDPCRD
```

Comment:
This should be idp.GetIDPName(), title is a friendly name. If title is not set by the config, it gets set as name during the config initalization/validation

Reply:
ah, yes...  you're right.  Not sure why I put title there.